modules/test/playwright/fixtures/clientExtensionsPageTest.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {test} from '@playwright/test';
+
+import {ClientExtensionsPage} from '../pages/product-navigation-applications-menu/ClientExtensionsPage';
+
+const clientExtensionsPageTest = test.extend<{
+	clientExtensionsPage: ClientExtensionsPage;
+}>({
+	clientExtensionsPage: async ({page}, use) => {
+		const clientExtensionsPage = new ClientExtensionsPage(page);
+
+		await clientExtensionsPage.goto();
+
+		await use(clientExtensionsPage);
+	},
+});
+
+export {clientExtensionsPageTest};
```

Comment:
I applied naming conventions to files that we [discussed in Slack](https://liferay.slack.com/archives/C06A59K5D46/p1704885957892329). I didn't apply it to existing tests, I can do it in a dedicated task.

Reply:
Oh, I forgot about that discusion. Then all my comments in https://github.com/liferay-frontend/liferay-portal/pull/3924 regarding naming are wrong :face_with_head_bandage:  (CC: @dsanz @chestofwonders)

However, unless we first rename everything to follow the conventions and send it to Brian, he will most probably refuse having two naming conventions merged to master... :shrug: